pkg/controller/mcmhub/mcmhub_controller.go

```diff
@@ -550,7 +550,7 @@ func (r *ReconcileSubscription) Reconcile(request reconcile.Request) (result rec
 				}
 			}
 		}
-	} else { //local: true
+	} else if *(pl).Local { //local: true
```

Comment:
maybe its safer if you do this: `(pl).Local != nil && *(pl).Local`

Reply:
or like this to make it further safer :-)

if pl != nil && pl.Local != nil && *(pl).Local {
...
}

Reply:
Xiangjing the pl nill check is done here https://github.com/open-cluster-management/multicloud-operators-subscription/blob/master/pkg/controller/mcmhub/mcmhub_controller.go#L527